feegrant/index.md

```diff
@@ -0,0 +1,239 @@
+
+## Use Feegrant to Grant an Allowance
+
+An often discussed use case for the feegrant module is that blockchains or smart contracts can allow new users to start interacting with the chain or contract without having to acquire tokens first, greatly improving the onboarding experience.
+
+There are two [fee allowance types](https://docs.cosmos.network/v0.44/modules/feegrant/01_concepts.html#fee-allowance-types) that the feegrant module implements: `BasicAllowance` and `PeriodicAllowance`, the latter allows you to setup a periodic allowance to another user. In this tutorial, you will set up two tokens in your blockchain: one default token called `stake`, which will be used for fees, and another called `kudos`, for sending to your friends.
+
+You will learn how to spin up a single node network using the simulation application in Cosmos SDK (`simapp`) and set up Alice to be a validator. Then you'll have Bob send `kudos` tokens to Carol, while having zero `stake` to pay for fees. The latter will be possible because you'll let Carol grant a `BasicAllowance` to Bob.
+
+## Requirements
+
+Before you start the tutorial, you need to install the `simd` binary.
+
+Clone the `cosmos-sdk` repository:
+
+```bash
+git clone https://github.com/cosmos/cosmos-sdk
+```
+
+Change directories and check out `v0.44.0`:
+
+```bash
+cd cosmos-sdk && git checkout v0.44.0
+```
+
+Install the `simd` binary:
+
+```bash
+make install
+```
+
+Check to make sure the installation was successful:
+
+```bash
+simd version
+```
+
+You should see `0.44.0` printed to the console.
+
+## Configuration
+
+<!-- TODO: update tip to use `unsafe-reset-all` with better user experience -->
+
+::: tip
+If you have used `simd` before, you might already have a `.simapp` directory in your home directory. To keep the previous data, either save the directory to another location or use the `--home` flag and specify a different directory for each command in the following instructions. If you don't want to keep the previous data, remove the previous directory (`rm -rf ~/.simapp`).
+:::
+
+Run the following commands to configure the `simd` binary.
+
+Set the chain ID:
+
+```bash
+simd config chain-id demo
+```
+
+Set the [keyring backend](https://docs.cosmos.network/v0.42/run-node/keyring.html#the-test-backend):
+
+```bash
+simd config keyring-backend test
+```
+
+## Key Setup
+
+You'll have to create a few test keys for your users.
+
+Add a key for Alice, the validator:
+
+```bash
+simd keys add alice
+```
+
+Add a key for Bob, the grantee:
+
+```bash
+simd keys add bob
+```
+
+And lastly, add a key for Carol, the granter:
+
+```bash
+simd keys add carol
+```
+
+If you'd like to see an overview of your keys, use:
+
+```bash
+simd keys list
+```
+
+In order to avoid having to copy-paste their address all the time, you should export them to variables that you can keep using during this tutorial:
```

Comment:
```suggestion
::: tip
To avoid having to copy and paste the user addresses now is a good time to export the user keys to variables that you can access and use for this tutorial. 
:::
```